### ISSUE TITLE
Error while spawning vagrant cluster: destination path 'etcd' already exists and is not an empty directory

### ISSUE THREAD

**Issue by akram** [openshift/origin#765]
Hi,

when spanning a new cluster with vagrant up, or doing a vagrant provision we got the following error:
``` 
fatal: destination path 'etcd' already exists and is not an empty directory
```
This is due to the hack/install-etcd.sh script which fails when trying to do a git clone on a previous installation.

workaround is to do a make clean before launching vagrant, or add a check in the etcd-install.sh script:
```
cd "${OS_ROOT}/_output"

if [ ! -d "etcd" ]; then
  git clone https://github.com/coreos/etcd.git
fi

cd etcd
```

**Comment by mfojtik** [comment]
@goldmann I think we saw this as well

@akram can you make a PR with the fix?

**Comment by akram** [comment]
@mfojtik OK, here it is: #773

**Comment by akram** [comment]
supposed to be fixed on #773 , closing it
